orbit/pkg/update/update_test.go

```diff
@@ -50,15 +47,8 @@ func TestMakeRepoPath(t *testing.T) {
 	}
 
 	for _, tt := range testCases {
-		tt := tt
 		t.Run(tt.expected, func(t *testing.T) {
-			t.Parallel()
-
-			var opt Options
-			// Must deep copy DefaultOptions, otherwise there is a race condition when modifying the
-			// opt.Targets map in parallel tests below.
-			err := copier.CopyWithOption(&opt, DefaultOptions, copier.Option{DeepCopy: true})
-			require.NoError(t, err)
+			opt := DefaultOptions
```

Comment:
Test was failing on CI with:
```
--- FAIL: TestMakeRepoPath (0.00s)
    --- FAIL: TestMakeRepoPath/osqueryd/linux/4.6.0/osqueryd (0.00s)
panic: reflect.Value.Addr of unaddressable value [recovered]
	panic: reflect.Value.Addr of unaddressable value

goroutine 39 [running]:
testing.tRunner.func1.2({0x7c7820, 0x8e59c0})
	/opt/hostedtoolcache/go/1.18.2/x64/src/testing/testing.go:1389 +0x24e
testing.tRunner.func1()
	/opt/hostedtoolcache/go/1.18.2/x64/src/testing/testing.go:1392 +0x39f
panic({0x7c7820, 0x8e59c0})
	/opt/hostedtoolcache/go/1.18.2/x64/src/runtime/panic.go:838 +0x207
reflect.Value.Addr({0x7bf4c0?, 0xc000311498?, 0x7dc7a0?})
	/opt/hostedtoolcache/go/1.18.2/x64/src/reflect/value.go:273 +0x65
github.com/jinzhu/copier.copier({0x7ba500?, 0xc00043cee8}, {0x805e20?, 0xc00010b4d0?}, {0x38?, 0xcf?})
	/home/runner/go/pkg/mod/github.com/jinzhu/copier@v0.3.2/copier.go:134 +0x1385
github.com/jinzhu/copier.copier({0x7b9fc0?, 0xc00043ceb0}, {0x81ff20?, 0xc00043cf00?}, {0x0?, 0x1?})
	/home/runner/go/pkg/mod/github.com/jinzhu/copier@v0.3.2/copier.go:253 +0x24b2
github.com/jinzhu/copier.CopyWithOption(...)
	/home/runner/go/pkg/mod/github.com/jinzhu/copier@v0.3.2/copier.go:57
github.com/fleetdm/fleet/v4/orbit/pkg/update.TestMakeRepoPath.func1(0xc0001cc4e0?)
	/home/runner/work/fleet/fleet/orbit/pkg/update/update_test.go:60 +0x85
testing.tRunner(0xc0001cc9c0, 0xc00043cc80)
	/opt/hostedtoolcache/go/1.18.2/x64/src/testing/testing.go:1439 +0x102
created by testing.(*T).Run
	/opt/hostedtoolcache/go/1.18.2/x64/src/testing/testing.go:1[48](https://github.com/fleetdm/fleet/runs/6668534941?check_suite_focus=true#step:8:49)6 +0x35f
```
Not really necessary to run this specific test in parallel.